apps/rush-lib/src/api/Rush.ts

```diff
@@ -99,6 +100,15 @@ export class Rush {
     return this._version!;
   }
 
+  /**
+   * Assign the RUSH_INVOKED_FOLDER environment variable during startup.
+   *
+   * @internal
+   */
+  public static _assignRushInvokedFolder(): void {
+    process.env[EnvironmentVariableNames.RUSH_INVOKED_FOLDER] = process.cwd();
```

Comment:
Can we either normalize this to use `/` as the directory separator 100% of the time, or else add an option to `rush.json` to do so? Not having to convert all the time is really helpful for speeding up code.

Reply:
Since this variable is for use by shell commands, doesn't it make sense for `RUSH_INVOKED_FOLDER` to use the operating system's native slashes?  For example `C:/folder/file.txt` is not actually a legal path on Windows, even though some APIs will magically interpret `/` as `\`.  For example if you do `dir C:/Folder/File.txt` it will fail.

Reply:
I thought it was for use by npm scripts?

Reply:
In any case, that argument falls under the "or else add an option" category. Having to convert paths at intake all the time gets tedious, since using `/` is the fully cross-platform way to do so (supported by the `fs` module, PowerShell Core, bash, ...), and also the only portable way to persist them to disk.
See recent issues with zip files being generated using `\` as a path separator, which means that when unpacked on Linux they create top level files with names like `path\to\foo.js` which are very much not what the user expected.

Reply:
@dmichon-msft  Agreed it would be more convenient, but I'm not sure it would be more intuitive.

Try this on Windows:

**package.json**
```
  . . .
  "scripts": {
    "test": "set"
  }
  . . .
```

and then run `npm run test`.

Every single environment variable has backslashes, not slashes.  Some examples:
```
APPDATA=C:\Users\Owner\AppData\Roaming
ComSpec=C:\WINDOWS\system32\cmd.exe
HOME=C:\Users\Owner
INIT_CWD=C:\my-project
NODE_EXE=C:\Program Files\nodejs\\node.exe
NPM_CLI_JS=C:\Program Files\nodejs\node_modules\npm\bin\npm-cli.js
npm_config_cache=C:\Users\Owner\AppData\Roaming\npm-cache
npm_config_tmp=C:\Users\Owner\AppData\Local\Temp
npm_config_userconfig=C:\Users\Owner\.npmrc
npm_node_execpath=C:\Program Files\nodejs\node.exe
NVM_HOME=C:\Users\Owner\AppData\Roaming\nvm
NVM_SYMLINK=C:\Program Files\nodejs
PSModulePath=C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
PUBLIC=C:\Users\Public
TEMP=C:\Users\Owner\AppData\Local\Temp
USERPROFILE=C:\Users\Owner
windir=C:\WINDOWS
```

This includes `INIT_CWD` which is NPM's own equivalent of `RUSH_INVOKED_FOLDER`.